src/core_write.cpp

```diff
@@ -186,6 +187,10 @@ void TxToUniv(const CTransaction& tx, const uint256& hashBlock, UniValue& entry,
     entry.pushKV("locktime", (int64_t)tx.nLockTime);
 
     UniValue vin(UniValue::VARR);
+
+    CAmount nTotalIn = 0;
+    CAmount nTotalOut = 0;
```

Comment:
Why did you revert these back to the old variable name style? (I had changed them to `amt_total_in`/`amt_total_out` in my rebase...)

Reply:
No particular reason; I'd be happy with either way.